docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -1169,16 +1152,16 @@ cy
   .click()
 ```
 
-Cypress will automatically _wait_ for elements to pass their default assertions.
-Like with the explicit assertions you've added, all of these assertions share
-the _same_ timeout values.
+Cypress will automatically _wait_ for elements to pass their implicit
+assertions. Like with the explicit assertions you've added, all of these
+assertions share the _same_ timeout values.
```

Comment:
Maybe it'd be nice to mention where these timeout values come from.

Reply:
Rewrote these sentences.

> Cypress will automatically _wait_ for elements to pass their implicit assertions. See [Timeouts](#Timeouts) below for more on how timeouts are determined.